bridge-token-factory/build.sh

```diff
@@ -11,7 +11,7 @@ docker run \
      -w /host/bridge-token-factory \
      -e RUSTFLAGS='-C link-arg=-s' \
      nearprotocol/contract-builder \
-     cargo +stable build --target wasm32-unknown-unknown --release
+     /bin/bash -c "rustup target add wasm32-unknown-unknown; cargo build --target wasm32-unknown-unknown --release"
```

Comment:
Why is this needed?

Reply:
`wasm32-unknown-unknown` is not installed in the [current docker image](https://hub.docker.com/r/nearprotocol/contract-builder) for the latest toolchain. Notice it haven't been updated for 8 months. The [Dockerfile](https://github.com/near/near-sdk-rs/blob/master/contact-builder/Dockerfile) is correct but we need to publish the new image. Filed issue at https://github.com/near/near-sdk-rs/issues/310

The error I'm getting right now:

```
   Compiling bridge-token-factory v0.1.0 (/host/bridge-token-factory)
error[E0463]: can't find crate for `std`
  |
  = note: the `wasm32-unknown-unknown` target may not be installed

error: aborting due to previous error

For more information about this error, try `rustc --explain E0463`.
error: could not compile `bridge-token-factory`.
```